solr/core/src/test/org/apache/solr/filestore/TestDistribPackageStore.java

```diff
@@ -68,12 +65,13 @@ public void teardown() {
     System.clearProperty("enable.packages");
   }
   
-  @SuppressWarnings({"unchecked"})
+  @SuppressWarnings({"unchecked", "rawtypes"})
```

Comment:
is this necessary?

Reply:
depends on which rig I'm on so probably not